ably/realtime/connection.py

```diff
@@ -210,7 +210,8 @@ async def send_protocol_message(self, protocolMessage):
 
     async def setup_ws(self):
         headers = HttpUtils.default_headers()
-        ws_url = f'wss://{self.options.get_realtime_host()}?key={self.__ably.key}'
+        ws_url = (f'wss://{self.options.get_realtime_host()}?key={self.__ably.key}'
```

Comment:
I think we should use `urllib.parse.urlencode` here now that we're sending more than one query param